wordpress-mobile/WordPressAuthenticator-iOS
# UL&S: Add the Site Address folder to the project

Closes #320 

Now that most of the base UI is ready for the Site Address flow, add a new Site Address folder to hold future UIViewControllers.

<img width="265" alt="Screen Shot 2020-07-08 at 3 41 21 PM" src="https://user-images.githubusercontent.com/1062444/86968480-da679080-c131-11ea-8699-1b92f358f3f3.png">

### To test
1. Check out the branch and open Xcode.

There is no companion PR for WPiOS since this is a minor update.
